src/frontend-scripts/components/section-main/Profile.jsx

```diff
@@ -19,7 +19,7 @@ class ProfileWrapper extends React.Component {
 		bioValue: '',
 		blacklistClicked: false,
 		blacklistShown: false,
-		openTime: Date.now()
+    openTime: Date.now()

```

Comment:
Indentation

```suggestion
		openTime: Date.now()
```

Reply:
I think this was due to how I was planning on doing something with the state but changed my mind. idk why github viewed this as a change>